Update Spanish translation (msg_hash_es.h).

ramiro opened the pull request:
Changes were generated by the following workflow:

1. .h -> .po conversion
2. Translation edit using a tool that supports gettext .po file format.
   e.g. poedit, dedicated modes of Emac, Vim, other editors
3. .po -> .h conversion

Steps 1 and 3 are handled by using WIP tool at

https://github.com/ramiro/RetroArch/blob/manage-i18n-with-gettext-tools/tools/intl.py

## Guidelines

1. Rebase before opening a pull request
2. If you are sending several unrelated fixes or features, use a branch and a separate pull request for each
3. If possible try squashing everything in a single commit. This is particularly beneficial in the case of feature merges since it allows easy bisecting when a problem arises


## Description

[Description of the pull request, detail any issues you are fixing or any features you are implementing]

## Related Issues

[Any issues this pull request may be addressing]

## Related Pull Requests

[Any other PRs from related repositories that might be needed for this pull request to work]

## Reviewers

[If possible @mention all the people that should review your pull request]

alfrix commented:
hi, i have been doing the translations so far, thanks for helping, usually i base msg_hash_es.h on msg_hash_us.h to not miss any changes, i have seen that you didn't translate some labels, is there a reason for that? i have left some words in English intentionally, for example FPS which led me to left Frames in english, also left Netplay and a few others.

alfrix commented on intl/msg_hash_es.h:
```diff
@@ -3843,7 +3976,7 @@ MSG_HASH(
 MSG_HASH(
 	MSG_FRAMES,
-	"Frames"
+	"Cuadros"
```
Please don't frames it's extensively used and linked with FPS

ramiro replied:
Ok. Thanks for the clarification.

alfrix commented on intl/msg_hash_es.h:
```diff
@@ -1,3 +1,131 @@
+MSG_HASH(
+	MENU_ENUM_LABEL_INPUT_DRIVER_LINUXRAW,
+	"linuxraw"
+	)
```
All the above 32 hashes don't appear in msg_hash_us.h, where did they come from? 
This breaks my method of diff against msg_hash_us.h, i have to remove them to get them to sync correctly

ramiro replied:
Actually these lines are from intl/msg_hash_lbl.h

I will fix the most obvious problems (the ones you commented on) and force push

I'm a bit shot of time right now but I have a longer WIP reply to the remaining points (one example of this are these 32 literals). Will post them in another PR hopefully soon.

alfrix commented on intl/msg_hash_es.h:
```diff
@@ -3011,7 +3144,7 @@ MSG_HASH(
 MSG_HASH(
 	MENU_ENUM_LABEL_VALUE_VIDEO_SWAP_INTERVAL,
-	"Vertical Sync (Vsync) Swap Interval"
+	"Intervalo de intercambio de sincroniz. vertical (Vsync)"
```
Keep labels short, sublabels can be as long as you want, but labels start to scroll left to right and looks awful
maybe a better one is
"Intervalo de intercambio de Vsync" 
(vsync it's under brackets in the upper term.)

ramiro replied:
Ok. Noted.

twinaphex commented:
Alright, let me know if this can be merged as-is.

alfrix commented:
merge i'll fix the issues later, in another PR

2018-03-19 9:51 GMT-03:00 Twinaphex <notifications@github.com>:

> Alright, let me know if this can be merged as-is.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/libretro/RetroArch/pull/6374#issuecomment-374200707>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABMIBXqOE441llStW5G_rwRyUJdOChMmks5tf6nYgaJpZM4SkIPD>
> .
>



-- 
Alfredo Monclus

ramiro commented:
Sorry for letting this without reacting sooner.

I've fixed the errors pointed by @alfrix and removed the other proposed changes.

I will open another PR when I've enhanced my tool with the ability to be more granular when filtering translatable literals to ignore (this is the reason there are 32 'extra' literals extracted from `msg_hash_lbl.h`.) when processing the RA source code.

PS: Gettext PO file fro which the submitted `msg_hash_es.h` was created is this one: https://github.com/ramiro/RetroArch/blob/8b176e8591891d5e7c083f722a311a161ad813aa/locale/es.po

Changes asked by @alfrix are: https://github.com/ramiro/RetroArch/commit/8b176e8591891d5e7c083f722a311a161ad813aa

alfrix commented:
@twinaphex this is ready

RobLoach commented:
Looks like there's a conflict.

twinaphex commented:
This needs to be rebased again unfortunately. After this is done, I will finally merge it. Sorry for the delay.

alfrix commented:
@twinaphex There is no need for this PR anymore, i committed his changes maintaining his authorship with 421c191
